src/main/java/org/broadinstitute/hellbender/engine/Walker.java

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+
+/**
+ * An abstract class to represent built-in walkers that inherit directly from {@link GATKTool}.
```

Comment:
Change to: 

"Base class for pre-packaged walker traversals in the GATK engine. Classes such as ReadWalker and VariantWalker in the engine package that implement a standardized traversal should extend this class rather than GATKTool. Actual concrete tool classes should extend one of the specific walker base types (such as ReadWalker) or GATKTool directly rather than this class.

Classes that extend Walker will be unable to directly access engine data sources unless they are in the engine package. This is to allow walker base classes such as ReadWalker direct datasource access while disallowing it for concrete walker tool implementations, which should get their data via apply()."

Reply:
Fixed.